airbyte-server/src/main/java/io/airbyte/server/RequestLogger.java

```diff
@@ -31,12 +31,11 @@
 public class RequestLogger implements ContainerRequestFilter, ContainerResponseFilter {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(RequestLogger.class);
+  private static final String REQUEST_BODY_PROPERTY = "requestBodyProperty";
```

Comment:
do you mind adding a javadoc comment to this class explaining how it works? e.g. by implementing both` ContainerRequestFilter` and `ContainerResponseFilter` it is able to grab the request body and then log the body as part of the response. It took me a long to remember how this was supposed to work and I theoretically already have context into it 🙃